src/main/resources/templates/login.html

```diff
@@ -0,0 +1,40 @@
+<html lang="en">
```

Comment:
should we use this as an opportunity to set up our front end? can be basic pending designs, but move basic html files there instead of having it here?

Reply:
Some of my thoughts, we can have login/logout pages as static assets in server side and have react app only started after user logged in, just like how different apps in rental manager works, they don't handle login logic. Or we can have the login/logout pages in react app like you suggested. Another thing we'll want to decide is if we want Google/FB single signon together with user/password register login, or only one of them.

Reply:
google/FB is tbd as we're considering only allowing military users to login for MVP

Reply:
hmmmm i've never experienced trying login/logout pages for static assets, no objections for that here